accessibility/index.html

```diff
@@ -0,0 +1,556 @@
+                </section>
+            
+                <!-- SUCCESS CRITERIA EXAMPLES END -->
+            
+             <!-- ERROR -->
+            
+            <section>
+                        <small>(SC 1.3.1 - SC 2.4.1 - SC 2.4.6 - SC 2.4.10)</small>
+                        <h2>Looks <span class="gdi">Complicated</span></h2>
+                        <h2>It Isn't!!</h2>
+						<small><a href="http://www.w3.org/TR/UNDERSTANDING-WCAG20/text-equiv.html" target="_blank">Understanding WCAG Sucess Criteria</a></small>
+					</section>
+                    <aside class="notes">
+				    How to easily use the WCAG Success Criteria - Understanding WCAG Sucess Criteria
+                    </aside>
+				</section>
+
+				<!-- Exercise 04 START-->
+
+				<section>
+					<h3>Exercise 4</h3> 
+					<p>Correct errors produced by automated test results</p>
+                    <small><a href="https://jsbin.com/liyewotacu/edit?html,css,output" target="_blank">Modify HTML from Exercise Two</a></small>		
+				</section>
+
+				<!-- Exercise 04 END -->
+
+				<!-- WHAT IS ARIA START -->
+				
+				<section>
+					<h3>Accessible Rich<br/>Internet Applications (ARIA 2.0)</h3>
+					<p>A set of accessibility attributes especially suited to HTML</p>		
+				</section>
+
+				<!-- WHAT IS ARIA END -->
+
+				<!-- ARIA ROLES START -->
+				
+				<section>
+					<h2>ARIA Roles</h2>
+                    <p><span class="gdi"><strong>Roles</strong></span> define the <strong>type</strong> of element or widget presented to the accessibiltiy API</p>
+                    <p>For example you can give a div a role="link" or role="button". The div will now be announced as a link or a button by a screen reader.</p>
+				</section>
+
+				<!-- ARIA ROLES START -->
+
+				<!-- Exercise 05 START -->
+
+				<section>
+					<h3>Exercise 5</h3>
+						<p align="center">1. Assign a varity of <strong>roles</strong> to a div</p>
+						<p align="center">2. Listen to changes in the screen reader</p>
+						<small><a href="https://jsbin.com/ranoxugeqo/edit?html,output" target="_blank">Try It</a></small>
+				</section>
+
+				<!-- Exercise 05 END -->
+        
+                <!-- More Information on ARIA Roles Start -->
+        
+                <section>
+					<h2>More Information on <span class="gdi">ARIA Roles</span></h2>
+        <small><a href="http://www.w3.org/TR/wai-aria/roles" target="_blank">The Roles Model</a></small>
+				</section>
+    
+                <!-- More Information on ARIA Roles End -->
+
+
+				<!-- ARIA STATES AND PROPERTIES START -->
+				
+				<section>
+                    <h2>ARIA <span class="gdi">States</span> and <span class="gdi">Properities</span></h2>
+					<p>States and Properities announce the state of an element to the accessibility API. Often times, but not always, states are activated by user interaction</p>
+                    <p>For example you can let a user know that a form input field is required by using aria-required="true"</p>
```

Comment:
Same thing with the `<code>` tag here. You can also add a whole code snippet below to illustrate a form input element.